index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="styles1.css" rel="stylesheet">
+        <title>Microverse First Project</title>
+    </head>
+    <body>
+        <header  class="toolbar">
+            <a href="#" class="text1">My Logo</a>
+            <nav>
+                <button class="nav" id="menu" type="button"><img src="Union.png" alt=""></button>
+            </nav>
+        </header>
+        
+        <div class="section">
+            <div class="headline">
+                <h1>I’m Tanmay Glad to see you!</h1>
+                <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="connect">LETS CONNECT</p>
+                <ul>
+                    <li><a href="#"><img src="Vector.svg" alt=""></a></li>
+                    <li><a href="#"><img src="Linkedinicon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(1).svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(2).svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(3).svg" alt=""></a></li>
```

Comment:
**Optional**

- [ ] I will also recommend you swap the last two `li's` so it could look exactly like the design arrangement.

Figma design:
![image](https://user-images.githubusercontent.com/51296741/143598597-fbe01626-2491-419d-a49d-b1c9bdcd20a2.png)

Your current design:
![image](https://user-images.githubusercontent.com/51296741/143598774-19355029-6877-485e-84f5-a336f0cc3998.png)